Makefile

```diff
@@ -0,0 +1,127 @@
+SHELL=/bin/bash -o pipefail
+
+ifndef APP_NAME
+$(error Please run "source environment" in the repo root directory before running make commands)
+endif
+
```

Comment:
Great suggestion, thanks, will implement.

Reply:
Actually, your error was not due to your AWS_PROFILE being set, but because you had no valid AWS credentials set at all (no default profile set). I'll add a more explicit check for this.

Reply:
Added check.